lms/djangoapps/discussion_api/permissions.py

```diff
@@ -71,7 +71,7 @@ def get_editable_fields(cc_content, context):
 
     # Thread fields
     if cc_content["type"] == "thread":
-        ret |= {"following"}
+        ret |= {"following", "read"}
```

Comment:
Shouldn't `read` be allowed on closed threads?  Otherwise, you can never declare a closed thread as read.